src/lib/components/input/input.svelte

```diff
@@ -13,7 +13,7 @@
 		dimension?: Dimension
 		layout?: Layout
 		unit?: string
-		error?: Snippet
+		error?: string[]
```

Comment:
If there are more than one error then I would call this errors, and if there are no errors then it would be an empty list.

```suggestion
		errors: string[]
```

Reply:
Actually, I don't like that this differs from diete.design. Maybe we should create snippets where we use this.

Reply:
Or maybe we should fix this in diete too

Reply:
I could imagine diete using multiple options there (`Snippet`, `string` or `string[]`) but I would definitely keep `Snippet` which allows error styling.